pepper-apis/src/main/resources/db-changes/seed/DDP-5102-add-arabic-portuguese-vietnamese.xml

```diff
@@ -0,0 +1,593 @@
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                   http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+    <changeSet author="yufeng" id="20201009-add-arabic-portuguese-vietnamese">
+        <insert tableName="language_code">
+            <column name="iso_language_code" value="ar"/>
+        </insert>
+        <insert tableName="language_code">
+            <column name="iso_language_code" value="pt"/>
+        </insert>
+        <insert tableName="language_code">
+            <column name="iso_language_code" value="vi"/>
+        </insert>
+    </changeSet>
+
+    <changeSet author="yufeng" id="20201009-update-activity-type-name-trans">
+        <insert tableName="i18n_activity_type_name_trans">
+            <column name="activity_type_id" valueComputed="(select activity_type_id from activity_type where activity_type_code = 'FORMS')"/>
+            <column name="language_code_id" valueComputed="(select language_code_id from language_code where iso_language_code = 'ar')"/>
+            <column name="translation_text" value="النماذج"/>
+        </insert>
+        <insert tableName="i18n_activity_type_name_trans">
+            <column name="activity_type_id" valueComputed="(select activity_type_id from activity_type where activity_type_code = 'FORMS')"/>
+            <column name="language_code_id" valueComputed="(select language_code_id from language_code where iso_language_code = 'pt')"/>
+            <column name="translation_text" value="Formulários"/>
+        </insert>
+        <insert tableName="i18n_activity_type_name_trans">
+            <column name="activity_type_id" valueComputed="(select activity_type_id from activity_type where activity_type_code = 'FORMS')"/>
+            <column name="language_code_id" valueComputed="(select language_code_id from language_code where iso_language_code = 'vi')"/>
+            <column name="translation_text" value="Mẫu"/>
+        </insert>
+        <update tableName="i18n_activity_type_name_trans">
+            <column name="translation_text" value="Formulaires"/>
+            <where>
+                activity_type_id = (select activity_type_id from activity_type where activity_type_code = 'FORMS')
+                and language_code_id = (select language_code_id from language_code where iso_language_code = 'fr')
+            </where>
+        </update>
+        <update tableName="i18n_activity_type_name_trans">
+            <column name="translation_text" value="Формы"/>
+            <where>
+                activity_type_id = (select activity_type_id from activity_type where activity_type_code = 'FORMS')
+                and language_code_id = (select language_code_id from language_code where iso_language_code = 'ru')
+            </where>
+        </update>
+        <update tableName="i18n_activity_type_name_trans">
+            <column name="translation_text" value="表单"/>
```

Comment:
@charlottedye I think there's overlap between languages for TestBoston and Prion. Please double-check these changes here for Prion.

Reply:
We don't actually have the Prion translations yet (and I can't read whichever language this is--Chinese?) so I figure if the translators give us something different, we can figure this out then.